metrics/sink_test.go

```diff
@@ -93,24 +99,43 @@ func TestTrendSink(t *testing.T) {
 			assert.Equal(t, 100.0, sink.Max)
 			assert.Equal(t, 54.0, sink.Avg)
 		})
+		t.Run("negative", func(t *testing.T) {
```

Comment:
Is it possible to have a "mixed" Trend, e.g., `-1,0,2`? If so, is itng a test case for it is good?